x-pack/plugins/ml/common/types/trained_models.ts

```diff
@@ -98,6 +98,7 @@ export type TrainedModelConfigResponse = estypes.MlTrainedModelConfig & {
    * Associated pipelines. Extends response from the ES endpoint.
    */
   pipelines?: Record<string, PipelineDefinition> | null;
+  origin_job_exists?: boolean;
```

Comment:
I don't think this need to be optional route param. We should always perform this check when retrieving the models as it's useful information.

Reply:
Updated in 7c06abd24a23cdfb98b34fba27bfdce6aee3e1ad